@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreslowScheduler.sqf

```diff
@@ -0,0 +1,27 @@
+/* ----------------------------------------------------------------------------
```

Comment:
Could it be possible to have just one scheduler function with two function `queues` that it worries about?  Rather than having this as two functions

Reply:
I would not know how to do such marvelous coding ideas